ardupilot/equipment/power/20010.BatteryCells.uavcan

```diff
@@ -0,0 +1,8 @@
+#
+# Battery cell voltages & resistances
+# Rate: set by parameter on smart battery (default off)
+#
+
+uint8 index 					  # Cell block index, index 0 corresponds to cells 0 - 23, index 1 cells 24 - 48, etc
+float16[<=24] voltages            # [mV]
+float16[<=24] resistances         # [m Ohm] cell resistances: NAN first element: field not provided

```

Comment:
I know I'm the one who suggested resistances.... but it should be its own message.

Then a user could change a parameter on the device to send it or not.

Reply:
removed